app/controllers/invitations_controller.rb

```diff
@@ -80,7 +80,7 @@ def resource_params
   # https://github.com/plataformatec/devise/blob/v2.2/app/controllers/devise_controller.rb#L99
   # for details :)
   def unsanitised_user_params
-    params.fetch(:user, {})
+    params.require(:user).permit(:name, :email, :organisation_id, :invitation_token, :password, :password_confirmation, :require_2sv, :role).to_h
```

Comment:
Worth checking the GDS style guide for this. Imagine this will cause a linting error

Reply:
Would be nice to break this into multiple lines for readability